kai/incident_store.py

```diff
@@ -0,0 +1,292 @@
+__all__ = ["Application", "IncidentStore"]
+
+import copy
+import os
+import pprint
+
+import yaml
+
+from kai.report import Report
+
+
+class Application:
+    def __init__(
+        self,
+        name,
+        report,
+        repo=None,
```

Comment:
In future we will probably need to update 'repo' so that we have flexibility for case where repo-A is the original and repo-B is the updated version.  Less of a concern when integrated in Konveyor, more that this flexibility will help us earlier with testing samples we collect.

Reply:
We don't need to update this in this PR, just sharing this note for thinking through/discussing.  We can adjust it later when it's obvious we need it.